python/lsst/ts/atmcssimulator/enums.py

```diff
@@ -59,53 +78,57 @@ class CommandKey(str, enum.Enum):
 
 
 class Event(str, enum.Enum):
-    ALLAXESINPOSITION = "allAxesInPosition"
-    ATMOUNTSTATE = "atMountState"
-    AZIMUTHBRAKE1 = "azimuthBrake1"
-    AZIMUTHBRAKE2 = "azimuthBrake2"
-    AZIMUTHDRIVE1STATUS = "azimuthDrive1Status"
-    AZIMUTHDRIVE2STATUS = "azimuthDrive2Status"
-    AZIMUTHINPOSITION = "azimuthInPosition"
-    AZIMUTHLIMITSWITCHCCW = "azimuthLimitSwitchCCW"
-    AZIMUTHLIMITSWITCHCW = "azimuthLimitSwitchCW"
-    AZIMUTHTOPPLEBLOCKCCW = "azimuthToppleBlockCCW"
-    AZIMUTHTOPPLEBLOCKCW = "azimuthToppleBlockCW"
-    DETAILEDSTATE = "detailedState"
-    ELEVATIONBRAKE = "elevationBrake"
-    ELEVATIONDRIVESTATUS = "elevationDriveStatus"
-    ELEVATIONINPOSITION = "elevationInPosition"
-    ELEVATIONLIMITSWITCHLOWER = "elevationLimitSwitchLower"
-    ELEVATIONLIMITSWITCHUPPER = "elevationLimitSwitchUpper"
-    M3DRIVESTATUS = "m3DriveStatus"
-    M3INPOSITION = "m3InPosition"
-    M3PORTSELECTED = "m3PortSelected"
-    M3ROTATORDETENTSWITCHES = "m3RotatorDetentSwitches"
-    M3ROTATORLIMITSWITCHCCW = "m3RotatorLimitSwitchCCW"
-    M3ROTATORLIMITSWITCHCW = "m3RotatorLimitSwitchCW"
-    M3STATE = "m3State"
-    NASMYTH1BRAKE = "nasmyth1Brake"
-    NASMYTH1DRIVESTATUS = "nasmyth1DriveStatus"
-    NASMYTH1LIMITSWITCHCCW = "nasmyth1LimitSwitchCCW"
-    NASMYTH1LIMITSWITCHCW = "nasmyth1LimitSwitchCW"
-    NASMYTH1ROTATORINPOSITION = "nasmyth1RotatorInPosition"
-    NASMYTH2BRAKE = "nasmyth2Brake"
-    NASMYTH2DRIVESTATUS = "nasmyth2DriveStatus"
-    NASMYTH2LIMITSWITCHCCW = "nasmyth2LimitSwitchCCW"
-    NASMYTH2LIMITSWITCHCW = "nasmyth2LimitSwitchCW"
-    NASMYTH2ROTATORINPOSITION = "nasmyth2RotatorInPosition"
-    POSITIONLIMITS = "positionLimits"
-    TARGET = "target"
+    """Enum containing all event names."""
```

Comment:
What do you mean with "low-level controller event IDs"? Something internal to ATMCS or SalObj event names or something different altogether?

Reply:
"low-level controller event IDs" just means the way the low-level controller identifies a particular event. It could be a string whose name matches a SAL event topic, or some other string, or an integer.

I didn't see any signs that you actually use the event data in the CSC. In which case it is just like telemetry in that it can be automatically processed and otherwise ignored. But I suppose command replies are just another form of event, so at least in that sense you need some kind of event-specific handlers. Still, that can be quite limited if it just needs to handle command replies.

Reply:
After our chat via Slack I created DM-39802 to eliminate the JSON schemas and the related enums.